src/components/VErrorSection/VNoResults.vue

```diff
@@ -1,35 +1,56 @@
 <template>
   <div class="no-results text-center md:text-left">
-    <h1 class="break-words text-4xl md:text-6xl">
-      {{ $t("no-results.heading", { query: query.q }) }}
+    <h1 class="heading-4 md:heading-2 break-words">
+      {{ $t("no-results.heading", { query: searchTerm }) }}
     </h1>
-    <h2
-      class="mt-10 text-base font-normal md:mt-16 md:text-3xl md:font-semibold"
-    >
+    <h2 class="description-regular md:heading-5 mt-10 md:mt-16">
```

Comment:
I would replace the spacing `10` and `16` with `4` (16px) in all breakpoints to reduce the gap between both headings